app/templates/components/global-navigation/search.hbs

```diff
@@ -109,7 +109,7 @@
             <li
               class="{{if (equal index selectedSuggestionIndex) "wds-is-selected"}} wds-global-navigation__search__suggestion"
               onmouseenter={{action (mut selectedSuggestionIndex) index}}
-              {{action "onSearchSuggestionClick" index}}
+              {{action "onSearchSuggestionClick" index preventDefault=(equal (equal searchScope "cross-wiki") false)}}
```

Comment:
I'd still check that condition in the action handler to not send request to current wiki if not needed

Reply:
Right now AFAIK, both things will happen: browser will redirect you to the link you click, but in the background unnecessary request (that will end up with 404 and response will not be handled) will be sent